scripts/sct_label_vertebrae.py

```diff
@@ -372,14 +375,16 @@ def main(args=None):
             im_label_c2c3 = detect_c2c3(im_data, im_seg, contrast, verbose=verbose_detect_c2c3)
             ind_label = np.where(im_label_c2c3.data)
             if not np.size(ind_label) == 0:
+                # Disc C2/C3 has value 3.
                 im_label_c2c3.data[ind_label] = 3
```

Comment:
This may be a related change. I remember this being slightly different earlier in the PR's history.